### PR TITLE
Add test for WpThemeMagicPropertiesClassReflectionExtension

### PR DESCRIPTION
**Pull request by IanDelMar**

### CODE REVIEW

**Comment by szepeviktor** [comment]
Thank you.

File: tests/WpThemeMagicPropertiesClassReflectionExtensionTest.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+namespace SzepeViktor\PHPStan\WordPress\Tests;
+
+use SzepeViktor\PHPStan\WordPress\WpThemeMagicPropertiesClassReflectionExtension;
+
+class WpThemeMagicPropertiesClassReflectionExtensionTest extends \PHPStan\Testing\PHPStanTestCase
+{
+    protected function setUp(): void
+    {
+        $this->broker = $this->createBroker();
```
Comment (szepeviktor) [inline tests/WpThemeMagicPropertiesClassReflectionExtensionTest.php:13]: Where does broker come from?

**Review by szepeviktor — CHANGES_REQUESTED** [review]

File: tests/WpThemeMagicPropertiesClassReflectionExtensionTest.php (hunk 1 above)
Comment (szepeviktor) [reply tests/WpThemeMagicPropertiesClassReflectionExtensionTest.php:16]: From Ian!

File: tests/WpThemeMagicPropertiesClassReflectionExtensionTest.php [hunk 2]
```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SzepeViktor\PHPStan\WordPress\Tests;
+
+use SzepeViktor\PHPStan\WordPress\WpThemeMagicPropertiesClassReflectionExtension;
+
+class WpThemeMagicPropertiesClassReflectionExtensionTest extends PHPStanTestCase
```
Comment (szepeviktor) [inline tests/WpThemeMagicPropertiesClassReflectionExtensionTest.php:9]:
```suggestion
class WpThemeMagicPropertiesClassReflectionExtensionTest extends \PHPStan\Testing\PHPStanTestCase
```

**Comment by szepeviktor** [comment]
🎵 Payment for the PR: Shamisen plunking 🇯🇵 https://www.youtube.com/watch?v=RcqO0zkyQRo

**Comment by IanDelMar** [comment]
Very generous! :dancers:

**Comment by szepeviktor** [comment]
... we could have a non-existent property - but do not worry about it

**Comment by IanDelMar** [comment]
Do you mean that `WP_Theme` does not necessarily have all of these properties? I var_dumped all  properties of a theme where I removes all info from style.css except the name of the theme. All properties get populated at least with an empty string or an empty array.

**Comment by IanDelMar** [comment]
But now I'm wondering why we don't add them via the stubs' functionMap.

**Comment by szepeviktor** [comment]
I've meant to add a line like
```
['WP_Theme', 'gluegun', false],
```

**Comment by IanDelMar** [comment]
https://github.com/szepeviktor/phpstan-wordpress/blob/2e08b58b0060b7f878244a36e5481ff51815bd43/tests/WpThemeMagicPropertiesClassReflectionExtensionTest.php#L25

**Comment by szepeviktor** [comment]
You see?? The PR is longer than 1 line so viktor is 😵 confuzed!

**Pull request merged**